### PR TITLE
Bugfix no default color time

### PR DESCRIPTION
**Pull request by ymollard**

### CODE REVIEW

File: pyluos/modules/color.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 class Color(Module):
     def __init__(self, id, alias, robot):
         Module.__init__(self, 'Color', id, alias, robot)
+        self._time = 0.2
```
Comment (nicolas-rabault) [inline pyluos/modules/color.py:8]: At module boot, `self._time` is 0. But when pyluos connect to it we don't know the current value.
I suggest to put `self._time` to `None`

File: pyluos/version.py [hunk 2]
```diff
@@ -1 +1 @@
-version = '1.1.0'
+version = '1.1.1'
```
Comment (nicolas-rabault) [inline pyluos/version.py:1]: We gonna start doing pre-release following a `+n` rule. Can you put `1.1.0+1`instead of `1.1.1`

**Review by nicolas-rabault — CHANGES_REQUESTED** [review]
Indeed you spotted a bug, tank you for your contribution.

We gonna start making pre-release of pyluos, I make some comment about it on you PR.

**Comment by ymollard** [comment]
Done is PR https://github.com/Luos-Robotics/pyluos/pull/57

**Pull request closed without merging**